csrc/flash_attn_ck/mha_bwd.cpp

```diff
@@ -191,7 +190,7 @@ fmha_bwd_args get_ck_fmha_bwd_args(const mask_info &mask,
                          static_cast<ck_tile::index_t>(mask.type),
                          p_dropout,
                          p_undrop,
-                         {drop_seed, drop_offset}};
+                         drop_seed_offset};
```

Comment:
do we still need the `variant<>` parameter?

Reply:
other user might need value version of dropout seed & offset